src/lib/storage/migration/1567674609659-reencrypt-seed.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * This migration re-encrypts the encryptedEntropy in the wallet database using
+ * the jolocom-lib (which uses node crypto) instead of the previously used
+ * crypto-js dependency
+ */
+
+import { MigrationInterface, QueryRunner } from 'typeorm/browser'
+import { reencryptWithJolocomLib } from 'src/lib/compat/cryptojs'
+
+async function getPassword(): Promise<string> {
+  let password: string | undefined
+  // at the time of writing this file, the encryption password is stored in
+  // react-native-keychain and accessesed through src/lib/keychain.ts, but code
+  // is reproduced here in case things change in the future
+
+  if (typeof navigator !== 'undefined' && navigator.product == 'ReactNative') {
+    const Keychain = require('react-native-keychain')
```

Comment:
Do we have to require this here (i.e. as opposed to at the start of the file)?

Reply:
This require fails completely on node because `react-native-keychain` is not transpiled to ES5 (it uses ES6 imports). When loaded on the browser it will have been transpiled by metro, but in the node context we are just running `ts-node` directly. I tried to configure it to not exclude `react-native-keychain` but failed and I don't think it matters really

Reply:
Got it, makes sense